app/api-v1/controllers/Attachment/index.js

```diff
@@ -1,5 +1,73 @@
+const { parseAccept } = require('../../../utils/parseAcceptHeader')
+const logger = require('../../../utils/Logger')
+
+const db = require('../../../db')
+
+const { createAttachmentFromFile, createAttachment } = require('../Attachment/helpers')
+const { InternalError, NotFoundError, BadRequestError, NotAcceptableError } = require('../../../utils/errors')
+
 module.exports = {
   getAll: async function () {
     return { status: 500, response: { message: 'Not Implemented' } }
   },
+  get: async (req) => {
+    const { id } = req.params
+    if (!id) throw new BadRequestError('missing params')
+
+    const [attachment] = await db.getAttachment(req.params.id)
+    if (!attachment) throw new NotFoundError('Attachment Not Found')
+    const orderedAccept = parseAccept(req.headers.accept)
+
+    for (const mimeType of orderedAccept) {
```

Comment:
This is naughty implementation, but does what it should. Will add to the backlog for DA PHUTURE

Reply:
why is this naughty?

Reply:
There a couple of reasons. I thnk we are under utilising helper method where all the heavy lifting if happening.

Well, it's just to me looks naughty or something might not do, but again I might not know some underlying  reasons